examples/styles/examples.css

```diff
@@ -13,25 +13,24 @@
  * limitations under the License.
  */
 
-/* @import url('https://fonts.googleapis.com/css?family=Rubik'); */
 @import url('https://fonts.googleapis.com/css?family=Rubik:400,500');
 @import url('https://fonts.googleapis.com/css?family=Roboto+Mono:400,500');
 
 :root {
   --color-red: #ff5252;
   --font-monospace: 'Roboto Mono', monospace;
   --border-style: 1px solid rgba(0,0,0,.15);
-  --grouping-title-margin-top: 60px;
+  --grouping-title-margin-top: 40px;
   --snippet-offset: 20px -20px;
   --pre-padding: 16px 20px;
-  --demo-title-padding: 60px 10% 40px 0;
+  --demo-title-padding: 24px 10% 24px 0;
   --heading-h3-max-width: 480px;
+  --heading-h1-margin-top: 40px;
   --browser-support-desc-margin-top: 40px;
   --browser-support-icons-margin-top: 16px;
-  --button-size: 56px;
-  --icon-size: 32px;
+  --button-size: 36px;
+  --icon-size: 28px;
   --header-height: 72px;
-  --poster-color: #455A64;
```

Comment:
Is `white` okay as the poster color for everything?

Reply:
That's a mistake. white it is. will fix.